ravenframework/TSA/ARMA.py

```diff
@@ -83,14 +84,31 @@ class cls.
                          \xmlNode{gaussianize} node preceding the \xmlNode{arma} node instead of this
                          option.
                          """, default=False)
-    specs.addSub(InputData.parameterInputFactory('SignalLag', contentType=InputTypes.IntegerType,
-                 descr=r"""the number of terms in the AutoRegressive term to retain in the
-                       regression; typically represented as $P$ in literature."""))
-    specs.addSub(InputData.parameterInputFactory('NoiseLag', contentType=InputTypes.IntegerType,
-                 descr=r"""the number of terms in the Moving Average term to retain in the
-                       regression; typically represented as $Q$ in literature."""))
+    specs.addParam('auto_select', param_type=InputTypes.BoolType, required=False,
+                   descr=r""" """, default=False)
+    specs.addSub(InputData.parameterInputFactory('P', contentType=InputTypes.IntegerListType,
```

Comment:
Should we add the description of auto_select? `desc=r""" """`

Reply:
yes, I missed that!